designer/server/src/main/scala/pl/touk/nussknacker/ui/services/NotificationApiHttpService.scala

```diff
@@ -0,0 +1,37 @@
+package pl.touk.nussknacker.ui.services
+
+import com.typesafe.config.Config
+import com.typesafe.scalalogging.LazyLogging
+import pl.touk.nussknacker.ui.api.NotificationApiEndpoints
+import pl.touk.nussknacker.ui.notifications.NotificationService
+import pl.touk.nussknacker.ui.process.ProcessCategoryService
+import pl.touk.nussknacker.ui.security.api.{AuthenticationResources, LoggedUser}
+
+import java.time.Instant
+import scala.concurrent.ExecutionContext
+
+class NotificationApiHttpService(
+    config: Config,
+    authenticator: AuthenticationResources,
+    getProcessCategoryService: () => ProcessCategoryService,
+    notificationService: NotificationService
+)(implicit executionContext: ExecutionContext)
+    extends BaseHttpService(config, getProcessCategoryService, authenticator)
+    with LazyLogging {
+
+  private val notificationApiEndpoints = new NotificationApiEndpoints(authenticator.authenticationMethod())
+
+  expose {
+//    Was before ->  Symbol("after").as[Instant].optional
+    val notificationsAfter = Some(Instant.now())
```

Comment:
in akka-http-based implementation we have:
```scala
parameter(Symbol("after").as[Instant].optional)
```

It's a declaration of an optional query parameter decoded as Instant. 

It seems that in Tapir-based implementation we should use sth like that:
```scala
query[Option[Instant]]("after")
```
in endpoint declaration.

Make sure that we use the same format for decoding from String to Instant (probably it's ISO 8601). We should mention that in Open API doc too.